docs/deploying.md

```diff
@@ -1,578 +1,152 @@
 <!--GITHUB
 page_title: Deploying a registry server
 page_description: Explains how to deploy a registry server
-page_keywords: registry, service, images, repository
+page_keywords: registry, service, images, repository, deploy
 IGNORES-->
 
-
 # Deploying a registry server
 
-This section explains how to deploy a Docker Registry either privately
-for your own company or publicly for other users. For example, your company may
-require a private registry to support your continuous integration (CI) system as
-it builds new releases or test servers. Alternatively, your company may have a
-large number of products or services with images you wish to serve in a branded
-manner.
+You obviously need to [install Docker](https://docs.docker.com/installation/) (remember you need **Docker version 1.6.0 or newer**).
+
+## Getting started in 2 lines
+
+Create a folder for your registry data:
+
+     $ mkdir registry-data
+
+Start your registry:
+
+     $ docker run -d -p 5000:5000 \
+     		-v `pwd`/registry-data:/tmp/registry-dev \
```

Comment:
Can you clarify what the problem with `pwd` is on mac?
Also, I would like to avoid people getting into permissions issues that would bite them with non user paths.